libdevcore/CMakeLists.txt

```diff
@@ -9,7 +9,13 @@ target_include_directories(devcore SYSTEM PUBLIC ${BOOST_INCLUDE_DIR})
 add_dependencies(devcore BuildInfo.h)
 
 if (NOT EMSCRIPTEN)
-    find_package(LevelDB)
-    target_include_directories(devcore SYSTEM PUBLIC ${LEVELDB_INCLUDE_DIRS})
-    target_link_libraries(devcore Boost::Filesystem Boost::Random Boost::Thread ${LEVELDB_LIBRARIES})
+    if (ROCKSDB)
```

Comment:
Can you separate common dependencies from conditional leveldb / rocksdb ones? Also `if (NOT EMSCRIPTEN)` can be remove as it is not needed any more.